src/main/resources/project/project.xml

```diff
@@ -9750,9 +9750,9 @@
             </formalParameter>
 
             <formalParameter>
-                <formalParameterName>syncCells</formalParameterName>
+                <formalParameterName>syncActiveNodes</formalParameterName>
                 <defaultValue/>
-                <description>If checked, syncCells will be triggered after deployment.</description>
+                <description>If checked, syncActiveNodes will be triggered after deployment.</description>
```

Comment:
More verbose documentation required here, like "... if checked, the active nodes will be synchronized after the deployment. The flag is ignored for standalone server installation."